features/paint-order.yml

```
@@ -0,0 +1,7 @@
+name: paint-order
+description: "The `paint-order` CSS property sets the z-order of strokes, fills, and (in SVG content) markers. For example, `paint-order: markers stroke fill;` draws the markers, then stroke, then fill."
```

Continuing from https://github.com/web-platform-dx/web-features/pull/1696#discussion_r1750725748

I see what you mean about the implicit metaphor. What about this?

```suggestion
description: "The `paint-order` CSS property sets the z-order of strokes, fills, and (in SVG content) markers. For example, `paint-order: markers stroke fill;` draws the markers, then stroke on top of markers, then fill on top of both markers and stroke."
```